src/slack/slack-bot.js

```diff
@@ -206,6 +206,21 @@ export class SlackBot extends Bot {
       .then(() => Promise.delay(this.postMessageDelay));
   }
 
+  // Get the bot's name and a list of aliases suitable for use in a top-level
+  // command message handler.
+  getBotNameAndAliases() {
+    const {activeUserId, dataStore} = this.slack.rtmClient;
```

Comment:
Yeah, it's pretty tightly coupled for now. We're using `@slack/client` `rtmClient` and `webClient` APIs all over the place.